samples/js/node/optical_character_recognition/optical-character-recognition.js

```diff
@@ -0,0 +1,420 @@
+const { addon: ov } = require('openvino-node');
+const fs = require('node:fs');
+const path = require('node:path');
+const { createCanvas, ImageData } = require('canvas');
+const { cv } = require('opencv-wasm');
+const {
+  transform,
+  getImageData,
+  argMax,
+  setShape,
+} = require('../helpers.js');
+
+// Parsing and validation of input arguments
+if (process.argv.length !== 6)
```

Comment:
Shouldn't be called when require this file

Reply:
Hi @vishniakov-nikolai I have made all the changes proposed by you except of the last one , can you please elaborate on this **Shouldn't be called when require this file** ? 

With Regards
Prakash

Reply:
If you include this file from another script, like `const ocr = require('./optical-character-recognition.js')`, then you will get Error, because this if block will execute, but process argv will empty.

Reply:
Hi @vishniakov-nikolai thanks for explaining , I have made the changes.